<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewSettingsDialog</class>
 <widget class="QDialog" name="ViewSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>196</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>View Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="chkShowPoints">
     <property name="text">
      <string>Show Points</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkShowNormals">
     <property name="text">
      <string>Show point Normals</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkShowMesh">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Show Mesh</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkWireframe">
     <property name="text">
      <string>Show Wireframe</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ScalarControlWidget" name="pointSizeControl" native="true"/>
   </item>
   <item>
    <widget class="ColorSwatch" name="swatchPointColor" native="true">
     <property name="label" stdset="0">
      <string>Point Color</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ColorSwatch" name="swatchMeshColor" native="true">
     <property name="label" stdset="0">
      <string>Mesh Color</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ColorSwatch" name="swatchWireColor" native="true">
     <property name="label" stdset="0">
      <string>Wireframe Color</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ScalarControlWidget" name="cameraSpeedControl" native="true"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScalarControlWidget</class>
   <extends>QWidget</extends>
   <header>src/UI/ScalarControlWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ColorSwatch</class>
   <extends>QWidget</extends>
   <header>src/UI/ColorSwatch.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ViewSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
